public/app/percona/add-instance/components/AzureDiscovery/Discovery.styles.ts

```diff
@@ -0,0 +1,11 @@
+import { stylesFactory } from '@grafana/ui';
+import { css } from 'emotion';
+
+export const getStyles = stylesFactory(() => ({
```

Comment:
Grafana say the the hook is more preferable because it is memoized, and stylesFactory re-renders unnecessarily.